index.js

```diff
@@ -124,6 +125,25 @@ module.exports = class Kernel {
     }
   }
 
+  createHandler (create) {
+    // Inject metering
+    let code = this.callHandler({ to: meteringContract, data: create.data }).returnValue
+
+    let address = Utils.newAccountAddress(create.from, code)
```

Comment:
`Utils` is easy to confuse with to `util = require('util')`. t least for me :P  maybe move to kernel-util? A